sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/ConnectionString/ConnectionStringParser.cs

```diff
@@ -6,8 +6,8 @@
 using System.Linq;
 #if AZURE_MONITOR_EXPORTER
 using Azure.Monitor.OpenTelemetry.Exporter.Internals.Diagnostics;
-#elif LIVE_METRICS_EXPORTER
-using Azure.Monitor.OpenTelemetry.LiveMetrics.Internals.Diagnostics;
+#elif ASP_NET_CORE_DISTRO
```

Comment:
Does this change, break existing live metrics project?

Reply:
Yes this will break.
This change assumed we would delete the LiveMetrics project. Let me clean this up.